src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -361,17 +362,16 @@ public void testDeleteDirOnExit() throws IOException {
         //This just tests that the code runs without crashing.
         //It runs at jvm shutdown so there isn't a good way to test it properly.
         //If you see a directory in the hellbender main folder called
-
-        final File dir = new File(GATKBaseTest.publicTestDir + "I_SHOULD_HAVE_BEEN_DELETED");
-        IOUtils.deleteRecursivelyOnExit(dir);
+        final File dir = new File( "I_SHOULD_HAVE_BEEN_DELETED");
```

Comment:
Do you really want to create this file in the working directory?

Reply:
it's more visible that way.... but I can move it back to where it was...